_includes/project_cards.html

```diff
@@ -34,7 +34,7 @@
             <tr>
               <th>{{ project.name }}</th>
               <td>{{ project.description }}</td>
-              <td><a href="{{ project.code }}" class="card-link"><i class="fab fa-github fa-2x"></i></a></td>
+              <td><a href="{{ project.code }}" class="card-link"><i class="fab fa-{% if project.code contains "gitlab.com" %}gitlab{% else %}{% if project.code contains "pypi.org" %}python{% else %}github{% endif %}{% endif %} fa-2x"></i></a></td>
```

Comment:
@wtbarnes After I said that I looked at our projects and noticed one of them is hosted at pypi.org. What do you think of this nested conditional? It's not pretty but I can live with it. We'd be supporting icons for GitHub, GitLab, and PyPI. If the list grew any larger I'd probably reconsider this approach.

Reply:
If you're cool with this code it's ready to merge.